topics/metagenomics/tutorials/general-tutorial/tutorial.md

```diff
@@ -32,21 +32,18 @@ In the metagenomics fields, amplicon sequencing refers to capture and sequence o
 It can be 16S for bacteria or archea or 18S for eukaryotes.
 
 > ### {% icon tip %} Background: The 16S ribosomal RNA gene
-> ![The 16S ribosomal RNA gene](../../images/16S_gene.png) <br><br>
+> ![The 16S ribosomal RNA gene](../../images/16S_gene.png "The 16S ribosomal RNA gene. Credit: https://www.slideshare.net/beiko/ccbc-tutorial-beiko")
```

Comment:
Is link possible? Or simply as brackets?

Reply:
you can add a link but only as html and I wanted to avoid html 😄